### PR TITLE
dep: pull stable marshal fixes

### PR DESCRIPTION
**Pull request by kyessenov**
/assign @mandarjog 
/assign @philrud

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -345,8 +346,8 @@
     "pkg/util",
   ]
   pruneopts = "T"
-  revision = "eb553e72ac8724721a99ed85fcb116467c7a2bac"
-  version = "v0.6.7"
+  revision = "0ad6fa1cf0b9b6ca8f3617a7188a568e81f40b87"
```
Comment (kyessenov) [inline Gopkg.lock]: confirms latest envoy xDS is here

File: Gopkg.lock [hunk 2]
```diff
@@ -1643,7 +1644,7 @@
     "rbac/v1alpha1",
   ]
   pruneopts = "T"
-  revision = "e1557ace81b4f794d65ae2b769ebfc153e22a662"
+  revision = "0791ae64224c0c50dad4899d94c6d2e3f35d8d4a"
```
Comment (kyessenov) [inline Gopkg.lock]: confirms latest istio/api is here

**Comment by duderino** [comment]
@utka as soon as this is merged we can create RC4

**Comment by kyessenov** [comment]
Fixed some istioctl unit test that hardcoded text output.

**Comment by kyessenov** [comment]
@duderino Ready to be merged.

**Pull request merged**